idaes/models/properties/general_helmholtz/tests/test_functions_unit.py

```diff
@@ -1655,6 +1655,38 @@ def test_r1234ze_transport():
     )
 
 
+@pytest.mark.unit
+@pytest.mark.skipif(not available(), reason="General Helmholtz not available")
+def test_propane_transport():
+    m = pyo.ConcreteModel()
+    m.hparam = HelmholtzParameterBlock(
+        pure_component="propane", amount_basis=AmountBasis.MASS
+    )
+    te = HelmholtzThermoExpressions(m, m.hparam)
+
+    assert pytest.approx(78.97826, rel=1e-3) == pyo.value(
```

Comment:
Minor comment, but could we have a reference for these values? It would also be nice to have a few more data points to show that the functions are right.